src/main/java/com/google/impactdashboard/database_manager/data_read/DataReadManagerImpl.java

```diff
@@ -45,6 +46,24 @@ public DataReadManagerImpl() {
     return listOfProjects;
   }
 
+  /**
+   * Returns a list containing the identifying information of every organization
+   * present in the IAM Bindings table. 
+   */
+  public List<OrganizationIdentification> listOrganizations() {
+    QueryJobConfiguration queryConfiguration = queryConfigurationBuilder
+      .getOrganizationIdsConfiguration().build();
+    TableResult results = database.readDatabase(queryConfiguration);
+
+    List<OrganizationIdentification> listOfOrganizations = new ArrayList<>();
+    for (FieldValueList row : results.iterateAll()) {
```

Comment:
nit: seems like a good place to use stream() and collect(toList())?

Reply:
done!